lib/types.js

```diff
@@ -1,5 +1,13 @@
 // @flow
```

Comment:
Flow types were changed from `snake_case` to `camelCase` for a better JavaScript API.

Automatic translation from snake to camel is done via `lodash/snakeCase`.

Reply:
This is super clever!